website/docs/how-to/how-to-schedule-feature-releases.mdx

```diff
@@ -35,7 +35,7 @@ This guide assumes that you've got the following:
 
 To schedule a feature release via the UI:
 1. Add the desired activation strategy to the feature
-2. Open the constraint creator by using the "add constraint" button
+2. Open the constraint creator by using the "Add custom constraint" button
```

Comment:
Goes back to a previous comment: should we capitalize these? I'm a bit on the fence 💁🏼

```suggestion
2. Open the constraint creator by using the "add custom constraint" button
```